app/components/RegisterSingleTree/MapMarking.js

```diff
@@ -107,11 +105,12 @@ const MapMarking = ({ updateScreenState, inventoryState, resetRouteStack }) => {
       recenterCoords = [location.coords.longitude, location.coords.latitude];
     }
     setIsInitial(true);
-    camera && camera.current.setCamera( {
-      centerCoordinate: recenterCoords,
-      zoomLevel: 18,
-      animationDuration: 1000,
-    });
+    camera &&
+      camera.current.setCamera({
```

Comment:
We should add check for all the other usages of the camera as well (I got a problem registering sample trees):

<img width="554" alt="Bildschirmfoto 2021-03-12 um 10 37 13" src="https://user-images.githubusercontent.com/1532418/110921644-171e0400-831f-11eb-9780-78913898d153.png">